app/internal/event.py

```diff
@@ -0,0 +1,65 @@
+import logging
+import re
+
+from email_validator import validate_email, EmailSyntaxError
+from fastapi import HTTPException
+
+from starlette.status import HTTP_400_BAD_REQUEST
+
+from app.database.models import Event
+
+ZOOM_REGEX = re.compile(r'https://.*?\.zoom.us/[a-z]/.[^.,\b\s]+')
+
+
+def validate_zoom_link(location):
+    if ZOOM_REGEX.search(location) is None:
+        raise HTTPException(status_code=HTTP_400_BAD_REQUEST,
+                            detail="VC type with no valid zoom link")
+
+
+def get_invited_emails(invited_from_form):
+    invited_emails = []
+    for invited_email in invited_from_form.split(','):
+        invited_email = invited_email.strip()
+        try:
+            validate_email(invited_email, check_deliverability=False)
+            invited_emails.append(invited_email)
+        except EmailSyntaxError:
+            logging.debug(f'{invited_email} is not a valid email address')
```

Comment:
I remember you changed to error level :O

Reply:
true, rebased